### DISCUSSION TITLE
Color Picker with React Hook Form

### DISCUSSION THREAD

**Discussion by arndvs** [react-hook-form/react-hook-form#9204] [Q&A]
Hello, I'm curious if anyone has any examples of a color picker with React Hook Form? Thanks!

**Comment by bluebill1049** [comment]
i think it should be the same as a simple text input rigth?

**Comment by arndvs** [comment]
Hi Bill, thanks for the response. Great work on Hook Forms, it's really helpful  -thanks! 

To answer your question, I'm not entirely sure. Would it just be that easy? 

With a color picker I was thinking of something like this:
https://casesandberg.github.io/react-color/

or

https://github.com/omgovich/react-colorful

Like one of those drag and drop color pickers.

**Reply by bluebill1049** [reply]
Controller for external controlled components like you suggested above or register with native input color.
